### ISSUE TITLE
Dropping a procedure in beforeMigrate

### ISSUE THREAD

**Issue by austek** [flyway/flyway#2713]
###### Which version and edition of Flyway are you using?
`6.2.4`

###### If this is not the latest version, can you reproduce the issue with the latest one as well?
`N/A` using latest version

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)
`Java API`

###### Which database are you using (type & version)?
`MySQL 5.7` tried with both driver version `8.0.19`  and `5.1.47`

###### Which operating system are you using?
Mac OS Catalina 10.15.3

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, the SQL statement that failed (if relevant) and the command you ran.)
```
#-------------- FIX SCHEMA CHARSET AND COLLATION ------------------------
DROP procedure if exists update_schema_char_set;
DELIMITER $$

CREATE PROCEDURE update_schema_char_set()
  BEGIN
      IF (@@character_set_database != 'utf8mb4'OR @@collation_database != 'utf8mb4_unicode_ci') THEN
        ALTER DATABASE ${SM_CAL_DB_NAME} CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
      END IF;
  END $$
DELIMITER ;

CALL update_schema_char_set();
DROP procedure if exists update_schema_char_set;
```
###### What did you expect to see?
executes successfully 

###### What did you see instead?
```
org.flywaydb.core.api.FlywayException: Unable to parse statement in db/migration/beforeMigrate.sql at line 11 col 12: Incomplete statement at line 11 col 12: 

CALL update_schema_char_set();
DROP procedure if exists update_schema_char_set;
```

Also, commenting first `DROP procedure if exists update_schema_char_set;` seems to fix the problem

**Comment by juliahayward** [comment]
I believe this has been fixed in Flyway 6.3.0. Could you update and try again please?

**Comment by juliahayward** [comment]
(I think it's a duplicate of #2687)

**Comment by austek** [comment]
Can confirm fixed in 6.3.0
